libkbfs/disk_block_cache.go

```diff
@@ -83,16 +132,26 @@ func (cache *DiskBlockCacheStandard) getSizes() (blockSize int64, lruSize int64,
 	return blockSizes[0], lruSizes[0], nil
 }
 
-func (cache *DiskBlockCacheStandard) compactCaches(ctx context.Context) {
+// compactCachesLocked manually forces both the block cache and LRU cache to
+// compact their underlying data.
+func (cache *DiskBlockCacheStandard) compactCachesLocked(ctx context.Context) {
 	cache.blockDb.CompactRange(util.Range{})
 	cache.lruDb.CompactRange(util.Range{})
-	bSize, lruSize, err := cache.getSizes()
+	bSize, lruSize, err := cache.getSizesLocked()
 	cache.log.CDebugf(ctx, "Disk Cache bSize=%d lruSize=%d err=%+v", bSize, lruSize, err)
 }
 
+// lruKey generates an LRU cache key from a tlf.ID and a binary-encoded block
+// ID.
+func (_ *DiskBlockCacheStandard) lruKey(tlfID tlf.ID, blockKey []byte) []byte {
```

Comment:
This doesn't have to be a struct method, can just be a helper.  Not sure it matters though.